RecoLocalCalo/EcalRecProducers/plugins/EcalRecHitWorkerRecover.cc

```diff
@@ -278,7 +278,7 @@ bool EcalRecHitWorkerRecover::run(const edm::Event& evt,
       float tpEt = ecalScale.getTPGInGeV(tp->compressedEt(), tp->id());
       float tpEtThreshEB = logWarningEtThreshold_EB_FE_;
       if (tpEt > tpEtThreshEB) {
-        edm::LogWarning("EnergyInDeadEB_FE") << "TP energy in the dead TT = " << tpEt << " at " << ttDetId;
+        edm::LogInfo("EnergyInDeadEB_FE") << "TP energy in the dead TT = " << tpEt << " at " << ttDetId;
```

Comment:
just for my education, is there a reason why this can't be further demoted to `LogDebug` such that they are compiled away? In any case in a production environment the `LogInfo` messages are not registered, and there was recently an issue (see [46510](https://github.com/cms-sw/cmssw/issues/46510)) about excessive memory allocations from very frequent `LogInfo` calls.

Reply:
If we want to just disable the messages, this can be done by changing the `logWarningEtThreshold_EB/EE_FE` parameters in the HLT configuration to -1 [noted here](https://github.com/cms-sw/cmssw/blob/master/RecoLocalCalo/EcalRecProducers/plugins/EcalRecHitWorkerRecover.cc#L175 ).  Thomas suggests that this would be preferable, so that we don't even waste computing time calculating the energy. I can prepare a separate PR for that.

Reply:
Ok. If that's preferable, please open a [CMSHLT](https://its.cern.ch/jira/projects/CMSHLT/issues) ticket with the request and if agreed we'll take care of it in the HLT menu.